delfin/db/sqlalchemy/models.py

```diff
@@ -286,3 +287,115 @@ class FailedTask(BASE, DelfinBase):
     job_id = Column(String(36))
     deleted_at = Column(DateTime)
     deleted = Column(Boolean, default=False)
+
+
+class StorageInitiator(BASE, DelfinBase):
+    """Represents the storage initiator attributes."""
+    __tablename__ = 'storage_initiators'
+    id = Column(String(36), primary_key=True)
+    storage_id = Column(String(36))
+    name = Column(String(255))
+    description = Column(String(255))
+    wwn = Column(String(255))
+    status = Column(String(255))
+    native_storage_host_id = Column(String(255))
+    native_storage_initiator_id = Column(String(255))
+
+
+class StorageHost(BASE, DelfinBase):
+    """Represents the storage host attributes."""
+    __tablename__ = 'storage_hosts'
+    id = Column(String(36), primary_key=True)
+    storage_id = Column(String(36))
+    name = Column(String(255))
+    description = Column(String(255))
+    os_type = Column(String(255))
+    storage_initiators = Column(Text)
```

Comment:
Is there a poosibility to use Array type https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.ARRAY

Reply:
Yes, sqlalchemy supports ARRAY type. But only when it is used with PostgreSQL backend. We can not use with SQLite

![Selection_388](https://user-images.githubusercontent.com/30930862/122006858-f6a98e00-cdd4-11eb-96fa-5b7ec4a6920c.png)

![Selection_389](https://user-images.githubusercontent.com/30930862/122006837-f0b3ad00-cdd4-11eb-8cc3-60d91964b7a1.png)